app/models/restaurant.rb

```diff
@@ -0,0 +1,49 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                :integer          not null, primary key
+#  name              :string
+#  address           :text
+#  accepts_10bis     :boolean
+#  max_delivery_time :integer
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  coordinates       :json
+#  cuisine_id        :integer
+#
+
+class Restaurant < ApplicationRecord
+  require "json-schema"
```

Comment:
in a Rails project usually every gem you add to the gemfile is automatically included across the app. Try to remove this line and see if you code still works. 

If it doesn't please movee this line to be outside of the class:
```ruby
require 'json-schema'

class Restaurant < ApplicationRecord
  ...
```